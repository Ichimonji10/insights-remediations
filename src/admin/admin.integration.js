'use strict';

const { request, auth } = require('../test');

describe('admin', function () {
    describe('500', function () {
        test('internal', async () => {
            await request
            .get('/v1/admin/500')
            .set(auth.emptyInternal)
            .expect(500);
        });

        test('not internal', async () => {
            await request
            .get('/v1/admin/500')
            .set(auth.emptyCustomer)
            .expect(404);
        });
    });
});
