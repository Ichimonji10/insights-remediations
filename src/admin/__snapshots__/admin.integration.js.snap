// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`users list users wildcard 1`] = `
Array [
  Object {
    "account_number": "demo",
    "created_by": "demoUser",
    "playbook_count": "4",
  },
  Object {
    "account_number": "testReadSingle",
    "created_by": "testReadSingleUser",
    "playbook_count": "5",
  },
  Object {
    "account_number": "testStatus",
    "created_by": "testStatus",
    "playbook_count": "1",
  },
  Object {
    "account_number": "testWrite",
    "created_by": "testWriteUser",
    "playbook_count": "9",
  },
  Object {
    "account_number": "test",
    "created_by": "tuser@redhat.com",
    "playbook_count": "5",
  },
]
`;
